dockerfiles/Dockerfile.apisix.deb

```diff
@@ -0,0 +1,49 @@
+ARG image_base="ubuntu"
+ARG image_tag="bionic"
+ARG apisix_branch="2.2"
+ARG iteration="0"
+ARG apisix_repo="https://github.com/apache/apisix"
+
+FROM ${image_base}:${image_tag}
+
+RUN set -x \
+    && apt-get update \
+    && apt-get -y install wget curl git gnupg2 unzip \
+    && wget -qO - https://openresty.org/package/pubkey.gpg | apt-key add - \
+    && apt-get update \
+    && apt-get -y install software-properties-common \
+    && add-apt-repository -y "deb http://openresty.org/package/ubuntu $(lsb_release -sc) main" \
+    && apt-get update \
+    && apt-get install -y openresty gcc make openresty-openssl111-dev \
+    && wget https://github.com/luarocks/luarocks/archive/v3.4.0.tar.gz \
+    && tar -xf v3.4.0.tar.gz \
+    && cd luarocks-3.4.0 || exit \
+    && ./configure --with-lua=/usr/local/openresty/luajit > build.log 2>&1 || (cat build.log && exit 1) \
```

Comment:
We should not use LuaJIT for build.
Lua 5.1 should be used instead.

Reply:
fixed.